lib/system.nim

```diff
@@ -54,6 +54,16 @@ type
 
 include "system/basic_types"
 
+const
```

Comment:
Why this moving around?

Reply:
Because otherwise thise symbols are not available in dollars.nim afaict.
IMO NimVersion should be moved too then.

Reply:
You mean the stuff that was *literally* two lines above, so now this version stuff is *literally* 2000 lines apart? :facepalm:

Reply:
> IMO NimVersion should be moved too then.

it can't; `NimVersion` relies on what's defined in dollars, so these need to be split apart anyways.
Putting it to the top right after `int` is defined enables using branching on Nim version below that. The fact that it's 2000 lines apart makes only a cosmetic difference.